lib/Runtime/Library/JavascriptArray.h

```diff
@@ -116,6 +116,7 @@ namespace Js
         static uint32 const MaxArrayLength = InvalidIndex;
         static uint32 const MaxInitialDenseLength=1<<18;
         static ushort const MergeSegmentsLengthHeuristics = 128; // If the length is less than MergeSegmentsLengthHeuristics then try to merge the segments
+        static uint64 const MaxSafeInteger = 0x1FFFFFFFFFFFFF;  // 2^53-1
```

Comment:
Could we name this something a bit more descriptive? (e.g. size related, when we enforce)